src/helpers/companiDates.js

```diff
@@ -0,0 +1,30 @@
+const luxon = require('luxon');
+
+exports.CompaniDate = (...args) => companiDateFactory(exports._formatMiscToCompanyDate(...args));
+
+const companiDateFactory = _date => ({
+  _date,
+
+  isSame(miscTypeOtherDate, unit) {
+    const otherDate = exports._formatMiscToCompanyDate(miscTypeOtherDate);
+
+    return this._date.hasSame(otherDate, unit);
+  },
+});
+
+exports._formatMiscToCompanyDate = (...args) => {
+  if (!args.length) return luxon.DateTime.now();
+
+  if (args.length === 1) {
+    if (args[0] instanceof Object && args[0]._date && args[0]._date instanceof luxon.DateTime) return args[0]._date;
+    if (args[0] instanceof luxon.DateTime) return args[0];
+    if (args[0] instanceof Date) return luxon.DateTime.fromJSDate(args[0]);
+    if (typeof args[0] === 'string') return luxon.DateTime.fromISO(args[0]);
+  }
+
+  if (args.length === 2 && typeof args[0] === 'string' && typeof args[1] === 'string') {
+    return luxon.DateTime.fromFormat(args[0], args[1], { zone: 'utc' });
```

Comment:
Est-ce que tu as fait des tests sur cette methode avec les dates telles qu'elles sont en bdd ? Sur la doc luxon il est note `use this zone if no offset is specified in the input string itself. Will also convert the DateTime to this zone` je me demande ce qu'il se passe si un `offset` est specifié

Reply:
le test est avec le format reçu de la bdd. On reçoit quelque chose de la forme 2021-11-24T07:00:00.000Z (de type Date) depuis la bdd. tu peux m'envoyer le lien de la doc où c'est écrit ça ?

Reply:
Quand je regarde direct en bdd j'ai un `+00:00`, mais je suis en train de me dire que c'est peut etre l'affichage compass :) 
le lien de la doc : https://moment.github.io/luxon/api-docs/index.html#datetimefromformat

Reply:
oui c'est l'affichage compass si tu console.log dans l'api quelque chose qui vient de l'api c'est sous le format que je t'ai mis au dessus